distribution/changelog.txt

```diff
@@ -19,6 +19,7 @@
 - Fix: [#20778] [Plugin] Incorrect target api when executing custom actions.
 - Fix: [#20807] Tertiary colour not copied with small scenery.
 - Fix: [#20964] Crash when player connects to server with a group assigned that no longer exists.
+- Fix: [#20624] Scrolling text glitches after language is changed.
```

Comment:
Broke the sorting

Reply:
Oops, my bad. I thought there was a CI check for this. Should someone fix it in the next commit?

Reply:
We never merged it :'(
https://github.com/OpenRCT2/OpenRCT2/pull/20541